database/dump.sql

```diff
@@ -1,267 +0,0 @@
---
```

Comment:
It looks like this file got emptied, but you should be sure to export your database again.